### ISSUE TITLE
Unable to complete Athena Practice FHIR setup

### ISSUE THREAD

**Issue by Anurag-Asati** [GEHealthcare/apiserver.gehealthcare.com#310]
Facing issue while setting up Athena Practice FHIR along with v23 Athena Practice.

Installed v23 Athena Practice application on a new Windows Server 2022 VM and trying to use checkinasyst.com AD account as it is already on-boarded with Athena. 

However, while working on FHIR setup -
- Logged in with user: caalerts4@checkinasyst.com when prompted for Azure AD login.
- Clicked on "Configure" button, we're encountering below message -
- **Error** - Please register first using https://apps.pro.nsf.athenahealth.com/app/register?templateType=Factory 

Could you please help us with below queries -
- The VM - GECPS-V23 which we are using for setup is not in any domain. Is it mandatory for the server to be in a domain for FHIR setup?
- If yes, does the server need to be in same domain as the AD user?
- Any other setup which I may've missed resulting in above message?

The given URL in the message does not show any options to register when logged in with AD user - [caalerts4@checkinasyst.com](mailto:caalerts4@checkinasyst.com).

 Please let me know your inputs on the same.

**Comment by deepaktiwari29** [comment]
@Anurag-Asati 
As discussed over the call, please check with your Network Team to unblock this below URL.
https://apps.pro.nsf.athenahealth.com/app/register

**Comment by Anurag-Asati** [comment]
Hi @deepaktiwari29 
I've checked with IT team and the given URL is not blocked. I've also trying to access the URL from outside the office network and still getting the same error.

**Comment by deepaktiwari29** [comment]
@Anurag-Asati 
We have recently introduced to a new geo blocking policy. With the introduction of new policy, the above URL is now only accessible from USA networks and we are blocking all the calls from coming from any other region. 
Can you please try connecting to USA network or VPN network and then try running the API registration tool once again?

**Comment by deepaktiwari29** [comment]
Hello @Anurag-Asati, hoping we have answered your questions and you are able to proceed. Marking this issue as closed, if you have any further questions, please open a new ticket.
